### DISCUSSION TITLE
Transgender Option

### DISCUSSION THREAD

**Discussion by WolfDieterKattnig** [pH7Software/pH7-Social-Dating-CMS#864] [Q&A]
Wouldn't it be a step into the right direction, if we put a TV/TS option or even better a custom gender option into the software? I'd rather not snub our fellow citicens by ignoring their wishes and needs.

Or can I modify the software/database myself?

What do you think?

**Comment by mnz1988** [comment]
if you go to admin panel, you can create all kind of genders

**Reply by WolfDieterKattnig** [reply]
I know I am dumb, but in Which part of admin?
